app/src/main/java/com/einmalfel/podlisten/Provider.java

```diff
@@ -137,6 +137,10 @@ public long getMaxAge() {
   private static HelperV1 helper;
```

Comment:
@georgekankava 
Honestly, I don't think just "sonar has a rule for this" is enough reason for changes. Adding synchronized setters doesn't make code much better IMO. There is indeed an issue with these static variables, which could be solved by making them volatile (which is more succinct) or by making _**both**_ setters & getters synchronized.